src/commands/src/shared/next-gen-parse.ts

```diff
@@ -3,7 +3,10 @@ import { Job } from '../../../entities/job';
 import { getDirectory } from '../../../job/jobHandler';
 import { CliCommandResponse, ExecuteCommandError, executeCliCommand } from '../helpers';
 
-const RSTSPEC_FLAG = '--rstspec=https://raw.githubusercontent.com/mongodb/snooty-parser/latest/snooty/rstspec.toml';
+const SNOOTY_PARSER_VERSION = process.env['SNOOTY_PARSER_VERSION']
+  ? `v${process.env['SNOOTY_PARSER_VERSION']}`
+  : 'main';
+const RSTSPEC_FLAG = `--rstspec=https://raw.githubusercontent.com/mongodb/snooty-parser/${SNOOTY_PARSER_VERSION}/snooty/rstspec.toml`;
```

Comment:
Do we need the `RSTSPEC_FLAG` at all? I believe by default, the parser should just use whatever version of the rstspec it has

Reply:
Sorry, redundant comment. Just saw [the comment](https://github.com/mongodb/docs-worker-pool/pull/1058#issuecomment-2189754742) from the other PR.

Reply:
yup! removed it according to the same reason. and removed the ARG -> ENV changes as we won't need the SNOOTY_PARSER_VERSION specified